target/linux/ramips/image/mt7621.mk

```diff
@@ -50,6 +51,20 @@ define Build/iodata-factory
 	fi
 endef
 
+define Build/iodata-mstc-header
+  ( \
+    data_size_crc="$$(dd if=$@ ibs=64 skip=1 2>/dev/null | gzip -c | tail -c 8 | od -An -tx8 --endian little | tr -d ' \n')"; \
+    echo -ne "$$(echo $$data_size_crc | sed 's/../\\x&/g')" | \
+      dd of=$@ bs=8 count=1 seek=7 conv=notrunc 2>/dev/null; \
+  )
+  dd if=/dev/zero of=$@ bs=4 count=1 seek=1 conv=notrunc 2>/dev/null
+  ( \
+    header_crc="$$(dd if=$@ bs=64 count=1 2>/dev/null | gzip -c | tail -c 8 | od -An -N4 -tx4 --endian little | tr -d ' \n')"; \
```

Comment:
This line is the longest line in the file, I think we'd better split it.

Reply:
Indeed, I'll do it.

Reply:
Environment variable in Bash cannot stores null-bytes (0x00), so I fold the lines.